### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MysteriousRSA**
Added New Logo to the header
Updated format
Added proper # Headers
made it look less like HTM
Fixed some spelling mistakes

### CODE REVIEW

**Review by BigSpice — APPROVED** [review]
Love this to the moon and back

File: README.md [hunk 1]
```diff
@@ -1,167 +1,169 @@
...
+
+* Manage installed [Northstar](https://northstar.tf) mods
+
+* Downloading and installing [Northstar](https://northstar.tf) mods from a GitHub/GitLab repository
+
+* Installing downloaded [Northstar](https://northstar.tf) mods (.zip files)
+
+* Easilly isntall custom Wepaon/Pilot Skins
+
+* Easilly start a [Northstar](https://northstar.tf) Dedicated Server
```
Comment (ConnorDoesDev) [inline README.md:28]: *easily

File: README.md [hunk 2]
```diff
@@ -1,167 +1,169 @@
...
-<br /><br />
+* Easilly install, update and launch [Northstar](https://northstar.tf)
+
+* Manage installed [Northstar](https://northstar.tf) mods
+
+* Downloading and installing [Northstar](https://northstar.tf) mods from a GitHub/GitLab repository
+
+* Installing downloaded [Northstar](https://northstar.tf) mods (.zip files)
+
+* Easilly isntall custom Wepaon/Pilot Skins
```
Comment (ConnorDoesDev) [inline README.md:26]: *easily

File: README.md [hunk 3]
```diff
@@ -1,167 +1,169 @@
...
-<br />
-<br />
-• Manage dedicated [Northstar](https://northstar.tf) servers.
-<br /><br />
+## Features:
 
-_Suggestions/bug reports are highly appreciated. Please create a new [issue](https://github.com/BigSpice/VTOL/issues/new] and fill out the fields._
-<br />
-<br /><br />
+* Easilly install, update and launch [Northstar](https://northstar.tf)
```
Comment (ConnorDoesDev) [inline README.md:18]: *easily

File: README.md [hunk 4]
```diff
@@ -1,167 +1,169 @@
...
+* Installing downloaded [Northstar](https://northstar.tf) mods (.zip files)
+
+* Easilly isntall custom Wepaon/Pilot Skins
+
+* Easilly start a [Northstar](https://northstar.tf) Dedicated Server
+
+* [Thunderstore](https://northstar.thunderstore.io) Mod Browser
+
+
+### WIP Features:
```
Comment (ConnorDoesDev) [inline README.md:33]: you may want to change this to just "WIP" or "Work In Progress"

File: README.md [hunk 5]
```diff
@@ -1,167 +1,169 @@
...
+
+* Create custom servers using this application to fine tune setups.
+
+* Manage dedicated [Northstar](https://northstar.tf) servers.
+
+### Suggestions/Bug reports?
```
Comment (ConnorDoesDev) [inline README.md:39]: preferably keep this at `_Suggestions/bug reports are highly appreciated. Please create a new [issue](https://github.com/BigSpice/VTOL/issues/new] and fill out the fields._`
(remove the header)

File: README.md [hunk 6]
```diff
@@ -1,167 +1,169 @@
...
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446412-d10f1f7e-bbc2-4558-8e8b-033aae2af410.png)<br /><br />
-   Use this to verify your existing Northstar installation.
-   <br /><br />
-   _By using that, the application will download the latest [Northstar release](https://github.com/R2Northstar/Northstar/releases/latest)._
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446466-26d91f33-948d-4051-9247-c21b20328515.png)
 
-<br /><br />
+* This application has an auto-update Feature in the Update section
```
Comment (ConnorDoesDev) [inline README.md:48]: the wording is a bit off here

File: README.md [hunk 7]
```diff
@@ -1,167 +1,169 @@
...
+
+![image](https://user-images.githubusercontent.com/23240514/152447226-88662295-b7a9-4fd5-a958-dfc5ce59bd9c.png)
+
+### Configuration
+
+ * If the launcher fails to automatically find your Titanfall 2 installation, please select ``Browse`` on the Main page and _browse_ to where your Titanfall 2 installation is, and select the Titnafall2 folder
```
Comment (ConnorDoesDev) [inline README.md:54]: "Titnafall2"

File: README.md [hunk 8]
```diff
@@ -1,167 +1,169 @@
...
-<br /><br /><br /><br />
+   ![image](https://user-images.githubusercontent.com/23240514/152446386-4bb584c1-038a-4dd6-9588-ac3827884144.png)
 
-<h1> MOD BROWSER </h1>
 
-_The [Thunderstore](https://northstar.thunderstore.io) integration has been released in v1.1.7_<br />
-<br />
+### Verify [Northstar](https://northstar.tf)
+
+* Point the ``+Northstar install Exe Directory`` field to where northstar is installed:
```
Comment (ConnorDoesDev) [inline README.md:66]: preferably change this to `* Select the NorthstarLauncher.exe file in the Northstar installation field:`

File: README.md [hunk 9]
```diff
@@ -1,167 +1,169 @@
...
-<br />
+### Verify [Northstar](https://northstar.tf)
+
+* Point the ``+Northstar install Exe Directory`` field to where northstar is installed: 
+    *    Steam: `C:\Program Files (x86)\Steam\steamapps\common\Titanfall2NorthstarLauncher.exe`
+    *    Origin: `C:\Program Files (x86)\Origin Games\Titanfall2\NorthstarLauncher.exe` 
+
+   ![image](https://user-images.githubusercontent.com/23240514/152446412-d10f1f7e-bbc2-4558-8e8b-033aae2af410.png)
+   
+* To update or repair your northstar installation, Click the ``Update/Repair Northstar`` Button.
```
Comment (ConnorDoesDev) [inline README.md:72]: "Click" should be lowercase

File: README.md [hunk 10]
```diff
@@ -1,167 +1,169 @@
...
+    * This will download the latest [Northstar release](https://github.com/R2Northstar/Northstar/releases/latest) and repair/update your current instllation.
+
+
+   ![image](https://user-images.githubusercontent.com/23240514/152446466-26d91f33-948d-4051-9247-c21b20328515.png)
+
+_If an outdated version is detected, this will update it, If the installed version is up to date, it will repair it._
```
Comment (ConnorDoesDev) [inline README.md:78]: "If" should be lowercase

File: README.md [hunk 11]
```diff
@@ -1,167 +1,169 @@
...
+### Thunderstore Mod Browser
+
+_[Thunderstore](https://northstar.thunderstore.io) Support released as of version 1.1.7_
 ![image](https://user-images.githubusercontent.com/23240514/153077819-5cd04c1c-f280-453e-9d07-ecc08e8934e1.png)
-<br />
 
-Click "Refresh" to update the list of mods featured on Thunderstore.
-<br />
-<br />
+Click ``Refresh`` to Refresh the list of mods featured on Thunderstore.
```
Comment (ConnorDoesDev) [inline README.md:86]: "Refresh" should be lowercase

File: README.md [hunk 12]
```diff
@@ -1,167 +1,169 @@
...
+
 ![image](https://user-images.githubusercontent.com/23240514/152447664-09a21668-b54f-41df-8493-f1567c497ec1.png)
-<br /><br /><br /><br />
 
-<h1>Updating</h1>
 
-<br /><br />
+## Updating VTOL
+
+* You cam check for updates in the ``Updates`` Section
```
Comment (ConnorDoesDev) [inline README.md:135]: typo

**Review by ConnorDoesDev — CHANGES_REQUESTED** [review]
overall a pretty good lint, though has quite a few issues that need to be addressed.

File: README.md (hunk 6 above)
Comment (MysteriousRSA) [reply README.md:48]: What do you suggest I put there instead?

File: README.md (hunk 5 above)
Comment (MysteriousRSA) [reply README.md:39]: I'll change the text back, but I don't see the point in removing the header, why do you think it should be removed?

Also, there was a missing bracket on the link

File: README.md (hunk 7 above)
Comment (MysteriousRSA) [reply README.md:54]: What do you mean here?

File: README.md (hunk 12 above)
Comment (MysteriousRSA) [reply README.md:135]: well cock, i missed this one, thanks for telling me

File: README.md (hunk 5 above)
Comment (ConnorDoesDev) [reply README.md:39]: i see, you could change the header to "Contribute" or something

File: README.md [hunk 13]
```diff
@@ -1,167 +1,169 @@
...
 All settings persist between sessions.
-<br /><br />
 
-<h1>Troubleshooting</h1>
-If you see the following error message:
 
-<br /><br />
+# Troubleshooting
+* If you see one of the following error messages
+Simply click "Yes" and the application will attempt to fix the issue by downloading the required dependacies
```
[the remark this answers is not in the document]
Comment (MysteriousRSA) [reply README.md:160]: Alright, I'll change it back, thought a bit more description would help

File: README.md (hunk 6 above)
Comment (ConnorDoesDev) [reply README.md:48]: `You can preferably enable the automatic-updater module in the \`Update\` section`

File: README.md (hunk 7 above)
Comment (ConnorDoesDev) [reply README.md:54]: you have a typo in that

File: README.md (hunk 5 above)
Comment (MysteriousRSA) [reply README.md:39]: alright, updated the header

**Comment by BigSpice** [comment]
Good on my end after a proper slow review.

**Comment by MysteriousRSA** [comment]
changes made, any more feedback?

**Comment by BigSpice** [comment]
oop

File: README.md [hunk 14]
```diff
@@ -1,167 +1,169 @@
...
 
-_Installations will automatically be updated if an outdated version is detected, repairing will be done if the version is the same._
-<br /><br /><br /><br />
 
-<h1> MOD BROWSER </h1>
+### Verify [Northstar](https://northstar.tf)
 
-_The [Thunderstore](https://northstar.thunderstore.io) integration has been released in v1.1.7_<br />
-<br />
+* Select the NorthstarLauncher.exe file in the Northstar installation field:
```
Comment (BigSpice) [inline README.md]: Shouldnt this be check?

File: README.md (hunk 6 above)
Comment (MysteriousRSA) [reply README.md]: updated

File: README.md (hunk 7 above)
Comment (MysteriousRSA) [reply README.md]: got it

File: README.md [hunk 15]
```diff
@@ -1,167 +1,169 @@
...
 
-Click import skin and wait for the compatibliity check to finish.
-<br /><br />
-![image](https://user-images.githubusercontent.com/23240514/152447613-e3fc58d8-992e-46ad-9c2c-499a87888603.png)<br /><br />
-![image](https://user-images.githubusercontent.com/23240514/152447631-03075684-204e-4311-b3dc-4bd6e264d0f3.png)<br /><br />
-Once the compatibility is confirmed, the application can now inject the skin.
-<br /><br />
-Click "Install Skin" to proceed with the installation of the skin.
-<br /><br />
+* Click import skin and wait for the compatibliity check to finish.
```
Comment (ConnorDoesDev) [inline README.md:116]: typo in "compatibility"

**Review by ConnorDoesDev — CHANGES_REQUESTED** [review]
looks good. may want to fix that one issue though 😅

File: README.md (hunk 15 above)
Comment (MysteriousRSA) [reply README.md:116]: have I mentioned that I may be dumb

Comment (ConnorDoesDev) [reply README.md]: all good haha, these happen

**Review by ConnorDoesDev — APPROVED** [review]
alright. looks good!

**Comment by MysteriousRSA** [comment]
any more issues?

**Comment by BigSpice** [comment]
image
:|

**Comment by MysteriousRSA** [comment]
> 

what do you mean?

**Comment by BigSpice** [comment]
you technically cant select it, the field validates the exe
and gives the option to check for it again should it be missing at some point.

**Comment by MysteriousRSA** [comment]
alright, i'll update the wording there

**Comment by BigSpice** [comment]
image

**Review by BigSpice — APPROVED** [review]
Im gonna read more lmao

**Comment by MysteriousRSA** [comment]
cock i made a typo

**Comment by BigSpice** [comment]
gucchi

**Comment by MysteriousRSA** [comment]
Gucci*

**Comment by BigSpice** [comment]
merging

**Comment by BigSpice** [comment]
> Gucci*

damn

**Pull request merged**